### ISSUE TITLE
The rendered login/logout button text is reversed

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#892]
## Bug Report

**Problematic behavior**
The rendered text is reversed "Sophie Drouvroy connected as…”

**Expected behavior/code**
It would be more logical to have a voice restitution like this "Connected as … Sophie Drouvroy"

**Additional context/Screenshots**
<img width="1424" height="330" alt="Image" src="https://github.com/user-attachments/assets/c3a4aefd-0b40-4bab-ab22-ae074693b43c" />
<img width="247" height="84" alt="Image" src="https://github.com/user-attachments/assets/1eff33a2-03ea-41b2-ad13-cb177e0e8411" />

**Comment by lebaudantoine** [comment]
This was originally introduced by @manuhabitela during the project bootstrap phase.
If I recall correctly, the tooltip was intended purely as a visual hint to clarify the button’s purpose.

We should refactor it so it remains visual-only (not announced by screen readers) and instead provide an appropriate aria-label for accessibility.
